### PR TITLE
added metadata to standalone examples

### PR DESCRIPTION
**Pull request by Soot3**
I edited examples/interaction/ customjs_for_hover and customjs_for_selection

- [x] issues: #11765

### CODE REVIEW

**Review by Soot3 — COMMENTED** [review]
removed trailing spaces

File: examples/interaction/js_callbacks/customjs_for_hover.py [hunk 1]
```diff
@@ -1,3 +1,15 @@
+''' An interactive plot showcasing Bokeh's ability to add interactions
+using Custom Javascript.
+This example demonstrates adding links between points on a graph.
+These lines only show on hover using ``CustomJS`` callbacks to update the plot.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.segment, bokeh.plotting.figure.circle, bokeh.plotting.figure.add_tools, bokeh.models.HoverTool, bokeh.models.sources.ColumnDataSource, bokeh.models.callbacks.CustomJS
```
Comment (bryevdv) [inline examples/interaction/js_callbacks/customjs_for_hover.py]: The APIs line can get very long unless we keep it narrowly focuses, let's keep this one related to hover/callbacks
```suggestion
    :apis: bokeh.plotting.figure.add_tools, bokeh.models.HoverTool, bokeh.models.sources.ColumnDataSource, bokeh.models.callbacks.CustomJS
```

File: examples/interaction/js_callbacks/customjs_for_selection.py [hunk 2]
```diff
@@ -1,3 +1,16 @@
+''' An interactive plot showcasing Bokeh's ability to add interactions
+using Custom Javascript.
+This example demonstrates selecting highlighted points on a graph.
+These points will show on the secondary graph which uses ``CustomJS``
+callbacks to update its plot.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.circle, bokeh.layouts.row, bokeh.model.Model.js_on_change, bokeh.models.sources.ColumnDataSource, bokeh.models.callbacks.CustomJS
```
Comment (bryevdv) [inline examples/interaction/js_callbacks/customjs_for_selection.py]: similar observation as above
```suggestion
    :apis: bokeh.model.Model.js_on_change, bokeh.models.sources.ColumnDataSource, bokeh.models.callbacks.CustomJS
```

File: examples/interaction/js_callbacks/customjs_for_selection.py [hunk 3]
```diff
@@ -1,3 +1,16 @@
...
+    :refs: :ref:`ug_interaction_js_callbacks_customjs`
+    :keywords: hover, javascript callback, CustomJS
+'''
```
Comment (bryevdv) [inline examples/interaction/js_callbacks/customjs_for_selection.py:13]:
```suggestion

'''
```
blank line at the end to be consistent with existsing docstrings

File: examples/interaction/js_callbacks/customjs_for_hover.py [hunk 4]
```diff
@@ -1,3 +1,15 @@
...
+    :refs: :ref:`ug_interaction_js_callbacks_customjs`
+    :keywords: hover, javascript callback, CustomJS
+'''
```
Comment (bryevdv) [inline examples/interaction/js_callbacks/customjs_for_hover.py:12]:
```suggestion

'''
```
blank line at the end

**Review by bryevdv — CHANGES_REQUESTED** [review]
Thanks for the PR! Just a few very small comments

File: examples/interaction/js_callbacks/customjs_for_selection.py [hunk 5]
```diff
@@ -1,3 +1,16 @@
+''' An interactive plot showcasing Bokeh's ability to add interactions
+using Custom Javascript.
+This example demonstrates selecting highlighted points on a graph.
```
Comment (bryevdv) [inline examples/interaction/js_callbacks/customjs_for_selection.py]:
```suggestion
using Custom Javascript.

This example demonstrates selecting highlighted points on a graph.
```
maybe a blank line if this introduces a new paragraph

File: examples/interaction/js_callbacks/customjs_for_hover.py [hunk 6]
```diff
@@ -1,3 +1,15 @@
+''' An interactive plot showcasing Bokeh's ability to add interactions
+using Custom Javascript.
+This example demonstrates adding links between points on a graph.
```
Comment (bryevdv) [inline examples/interaction/js_callbacks/customjs_for_hover.py]:
```suggestion
using Custom Javascript.

This example demonstrates adding links between points on a graph.
```
similar blank line between paragraphs

**Review by bryevdv — CHANGES_REQUESTED** [review]

**Comment by Soot3** [comment]
I understand. Thank you

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks for the PR @Soot3 !

**Pull request merged**